cmd/argocd/commands/repo.go

```diff
@@ -236,6 +241,7 @@ func NewRepoAddCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 		},
 	}
 	command.Flags().BoolVar(&repoOpts.Upsert, "upsert", false, "Override an existing repository with the same name even if the spec differs")
+	command.Flags().BoolVar(&repoOpts.InsecureOCI, "insecure-oci", false, "Use http when accessing an OCI repository")
```

Comment:
Yes, the default is https, this instead forces the use of insecure http.

Reply:
I'd maybe go with `insecure-oci-force-http`, to help disambiguate in case we need to add another "insecure" flag like, `insecure-oci-skip-tls-verify` or something like that.

Reply:
Sounds good to me! Fixed